### DISCUSSION TITLE
Welcome to support Discussions!

### DISCUSSION THREAD

**Discussion by amit1rrr** [ReviewNB/support#84] [General]
## 👋 Welcome!

We’re using Discussions as a place to discuss optimal Jupyter workflow, challenges, ReviewNB feedback, roadmap & other questions that do not belong to `issues`. We hope that you ask questions you’re wondering about & Share ideas.

Welcome onboard! 😺

**Comment by josiahlaivins** [comment]
Hi! I'm really liking using reviewnb at our company. We have been combining it with nbdev!

We have 1 major-ish issue that I would be tempted to count as a bug. When conversations are resolved in reviewnb, they are not resolved in github. I personally consider this bug-ish because it makes "reviewing" difficult/unpleasant. I basically need to re-review the github conversations and resolve those separately. I would have expected that creating->commenting->resolving sonversations in reviewnb would sync with github.

Let me know what you opinion is, and I can create an issue!

**Comment by amit1rrr** [comment]
Hello @josiahlaivins 
Happy New Year! Glad you like using ReviewNB at your company.

> I would have expected that creating->commenting->resolving sonversations in reviewnb would sync with github.

100% agree. We would love to do this as well. The issue is, currently GitHub does not offer any API to "Resolve / Unresolve" a conversation. More [details here](https://github.com/ReviewNB/support/issues/66#issuecomment-671235089). 

We already have an open issue with GitHub support to add this missing API support. As soon as the API is available, we'll be using it. In the meantime, I'm sorry for a subpar experience!

Do let me know if there's ever anything else we can do to improve ReviewNB :)
